Druid/DRBytecodeGenerator.class.st

```diff
@@ -38,6 +38,15 @@ DRBytecodeGenerator >> generateJump: aDRBranch [
 { #category : 'API' }
 DRBytecodeGenerator >> generateTargetASTFromIR: aDRControlFlowGraph methodName: selector [
 	
+	self handleTemporaryVariable: aDRControlFlowGraph .
```

Comment:
What does it mean to "handle a temporary variable"?

Also, with this method name I'd expect that the argument is a temporary variable, not a cfg.